packages/manager/apps/dedicated/client/app/components/dedicated-cloud/dashboard/tiles/general-information/translations/Messages_fr_FR.json

```
@@ -1,8 +1,8 @@
 {
   "ovhManagerPccDashboardGeneralInformation_heading": "Informations générales",
 
-  "ovhManagerPccDashboardGeneralInformation_description_term": "Description",
-  "ovhManagerPccDashboardGeneralInformation_description_actionMenu_update": "Modifier la description",
+  "ovhManagerPccDashboardGeneralInformation_description_term": "Nom",
+  "ovhManagerPccDashboardGeneralInformation_description_actionMenu_update": "Modifier le nom",
```

It's better to rename also key names to keep link between keys and values
```suggestion
  "ovhManagerPccDashboardGeneralInformation_name_term": "Nom",
  "ovhManagerPccDashboardGeneralInformation_name_actionMenu_update": "Modifier le nom",
```